gps/app/locales/ht.json

```diff
@@ -1,82 +1,99 @@
 {
```

Comment:
this file has just been copied from `1.0.65`

Reply:
Same here, once you do a `yarn i18n:extract` (which only updates en.json) you'll need to compare keys with en.json.

Reply:
Ah very cool. Makes sense, doing that now.